src/packagedetails.cpp

```diff
@@ -42,7 +42,7 @@ PackageDetails::PackageDetails(OCI::Package *package,int count, int packagecount
 
     QObject::connect( meta, SIGNAL( finished( QString,QString ) ), this, SLOT( dataChanged( QString,QString ) ) );
 
-    m_showDescription = new QLabel( QString( "<a href = %1>Show Details</a>" ).arg( count ) );
+     m_showDescription = new QLabel( i18n("<a href = %1 style=\"text-decoration:none;\">Show Details</a>", count) );
```

Comment:
QString( "<a href = %1>%2</a>" ).arg( count ).arg( i18n("Show Details") )
so translators don't have to mess (and maybe break by accident) the markup.
Also, the style change wasn't mentioned in the commit log, so maybe it should be left for another pull request.